src/scene/materials/standard-material.js

```diff
@@ -34,6 +34,12 @@ Object.assign(pc, function () {
      * @property {boolean} specularVertexColor Use mesh vertex colors for specular. If specularMap or are specularTint are set, they'll be multiplied by vertex colors.
      * @property {string} specularVertexColorChannel Vertex color channels to use for specular. Can be "r", "g", "b", "a", "rgb" or any swizzled combination.
      *
+     * @property {boolean} enableGGXSpecular Enable GGX specular.
+     * @property {number} anisotropy Defines amount of anisotrophy.
+     * With anisotropy == 0, specular is isotropic
+     * With anisotropy < 0, anistropy direction aligns with the tangent, and specular anisotropy increases as the anisotropy value decreases to minimum of -1.
+     * With anisotropy > 0, anistropy direction aligns with the bi-normal, and specular anisotropy increases as anisotropy value increases to maximum of 1.
```

Comment:
I suspect these 3 'With' statement won't appear in the docs as you expect. I believe the new lines won't be preserved. One option is you use Markdown bullets (* before each line). But yeah, check by running `npm run docs` in your branch and check the output.

Reply:
Checked the docs and updated to make it clearer as suggested.